src/metaswitch/clearwater/queue_manager/test/__init__.py

```diff
@@ -30,15 +30,14 @@
 # under which the OpenSSL Project distributes the OpenSSL toolkit software,
 # as those licenses appear in the file LICENSE-OPENSSL.
 
-import logging
-import sys
-import os
 import random
+from metaswitch.common.logging_config import configure_test_logging
+import logging
+
+configure_test_logging()
```

Comment:
Again, can we get consistency in how logging config is imported and configured. 
Not essential, but would be nice